common/primitives/src/stateful_storage.rs

```diff
@@ -11,6 +11,8 @@ use utils::*;
 
 /// PageId is the unique identifier for a Page in Stateful Storage
 pub type PageId = u16;
+/// PageHash is the type/size of hash of the page content.
+pub type PageHash = u32;
```

Comment:
should we use `u32` or `[u8; 4]`?  Which one seems more compatible with other language implementations?

Reply:
> should we use `u32` or `[u8; 4]`? Which one seems more compatible with other language implementations?

👍

Reply:
Rather than block this PR, going to file a 1-point spike to answer this & also determine if the `TwoX32` hash is a valid selection given support in other languages.